modules/ROOT/nav.adoc

```diff
@@ -52,13 +61,14 @@
       **** xref:endpoint-sftp-send.adoc[SFTP Send Endpoint Settings]
     *** xref:as2-endpoints-algorithms.adoc[AS2 Signature and Encryption Algorithms]
     *** xref:create-keystore.adoc[Creating an AS2 Host Keystore]
-   ** xref:partner-manager-maps.adoc[Maps]
-  *** xref:create-map-inbound-edifact-json-xml.adoc[Create a Map for Inbound EDIFACT to JSON or XML]
-  *** xref:create-map-json-xml-to-outbound-edifact.adoc[Create a Map For XML or JSON to Outbound EDI EDIFACT]
-  *** xref:create-map-inbound-x12-json-xml.adoc[Create a Map for Inbound X12 to JSON or XML]
-  *** xref:create-map-json-xml-to-outbound-x12.adoc[Create a Map For XML or JSON to Outbound EDI X12]
-* xref:partner-manager-administration.[Partner Manager Administration]
- ** xref:activity-tracking.adoc[Track Transmission Activity]
+   ** xref:partner-manager-maps.adoc[Translation Maps]
+  *** xref:create-map-inbound-edifact-json-xml.adoc[Create a Translation Map for Inbound EDIFACT to JSON or XML]
+  *** xref:create-map-json-xml-to-outbound-edifact.adoc[Create a Translation Map For XML or JSON to Outbound EDI EDIFACT]
+  *** xref:create-map-inbound-x12-json-xml.adoc[Create a Translation Map for Inbound X12 to JSON or XML]
+  *** xref:create-map-json-xml-to-outbound-x12.adoc[Create a Translation Map For XML or JSON to Outbound EDI X12]
```

Comment:
Other topics were updated to use gerunds in the title (**Creating**), should these also follow the same rule to sync? According to the content templates, use gerunds in the title, and imperative verbs in the H2 task headers.